person.rb

```diff
@@ -18,4 +21,15 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission ? true : false
   end
+
+  def correct_name
+    @name
+  end
 end
+# #Check Using this code
+# person = Person.new(22, 'maximilianus')
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/208644158-1f7a1769-d900-4bd2-bb1a-a090de312077.png)


- [ ] It's great that you have a license file in your project but it looks like a template 🤔 Kindly add your **name**, **current year** and **app-type** 😊 to make your project licensed to you 👍